src/schemas/DropoutSchema.js

```diff
@@ -0,0 +1,54 @@
+module.exports = (conn) => {
+  const DropoutSchema = new conn.Schema({
+    city: {
+      type: String,
+      required: true,
+    },
+    cohort: {
+      type: String,
+      required: true,
+    },
+    user: {
+      type: conn.Schema.Types.ObjectId,
+      ref: 'User',
+      required: true,
+    },
+    stage: {
+      type: String,
+      required: true,
+    },
+    codeStudent: {
+      type: String,
+      required: true,
+    },
+    when: {
+      type: Date,
+      required: true,
+      default: Date.now,
+    },
+    type: {
+      type: String,
+      required: true,
+    },
+    reason: {
+      type: String,
+      required: true,
+    },
+    observations: {
+      type: String,
+      required: true,
+    },
+    sad: {
+      type: Boolean,
+      required: true,
+    },
+    otherReason: {
+      type: String,
+      required: true,
+    },
+    // createdAt??
+    // status??? (active, dropout, expelled??)
+    // ...
+  }, { collection: 'dropouts' });
```

Comment:
```suggestion
  }, { collection: 'dropouts', timestamps: true  });
```
ref: https://mongoosejs.com/docs/guide.html#timestamps, why? it could be too useful to know when a dropout was created and updated, in case we update the dropout to correct some fields